parlai/core/agents.py

```diff
@@ -375,21 +375,39 @@ def load_agent_module(opt):
     model_file = opt['model_file']
     optfile = model_file + '.opt'
     if os.path.isfile(optfile):
+        # Note: new_opt is a bit of a misnomer: opt is what comes from the CLI args,
```

Comment:
might there be a better name for it then? e.g., `saved_opt`?

Reply:
Agreeing with both @emilydinan and @klshuster here, might as well name this something more clearly referring to it being the saved opts associated with the model file. (`saved_opt` being a good choice)